modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -624,6 +650,157 @@ The output represents the transformed XML elements into JSON.
+            <Credential domain="NetworkID">
+                <Identity>TestIdentity</Identity>
+            </Credential>
+        </From>
+        <To>
+            <Credential domain="TEST">
+                <Identity>Identity</Identity>
+            </Credential>
+        </To>
+    </Header>
+</cXML>
+----
+
+==== Source
+
+[source,dataweave]
+----
+%dw 2.0
+output application/json
+---
+{
+    header: {
+        senderId: payload.cXML.Header.From.Credential.Identity,
+        receiverId: payload.cXML.Header.To.Credential.Identity,
+        docType: payload.^docType
+    }
+}
+----
+
+==== Output
+
+[source,json,linenums]
+----
+{
+  "header": {
+    "senderId": "Identity",
+    "receiverId": "TestIdentity",
+    "docType": {
+      "rootName": "cXML",
+      "systemId": "http://xml.cxml.org/schemas/cXML/1.2.014/cXML.dtd"
+    }
+  }
+}
+----
+
+[[example13]]
+=== Example: Write a `DTD` Value
+
+This example shows how to use the `docType` metadata to create a `DOCType` value in the XML output.
+
+==== Source
+
+[source,dataweave]
+----
+%dw 2.0
+output application/xml
+ns xml http://www.w3.org/XML/1998/namespace
+
+var cXmlObj = {
+  cXML @(payloadID: "9949494", xml#lang: "en-US", timestamp: "2002-02-04T18:39:09-08:00"): {
+      Header: {
+        From: {
+          Credential @(domain: "NetworkId"): {
+            Identity: "Identity"
+          }
+        }
+    }
+  }
+} as Object {
+    docType: { rootName: "cXML", systemId: "http://xml.cXML.org/schemas/cXML/1.2.014/cXML.dtd" }
+}
+---
+cXmlObj
+----
+
+==== Output
+
+[source,xml,linenums]
+----
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE cXML SYSTEM "http://xml.cXML.org/schemas/cXML/1.2.014/cXML.dtd">
+<cXML payloadID="9949494" xmlns:xml="http://www.w3.org/XML/1998/namespace" xml:lang="en-US" timestamp="2002-02-04T18:39:09-08:00">
+  <Header>
+    <From>
+      <Credential domain="NetworkId">
+        <Identity>Identity</Identity>
+      </Credential>
+    </From>
+  </Header>
+</cXML>
+----
+
+[[example14]]
+=== Example: Transform `DocType` Value to a String Representation
+
+This example uses https://docs.mulesoft.com/dataweave/latest/dw-dtd[Dtd Module (dw::xml::Dtd)] to transform
+a `DocType` value that includes a `systemId` to a string representation.
+
+==== Input
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE cXML SYSTEM "http://xml.cxml.org/schemas/cXML/1.2.014/cXML.dtd">
+<cXML xml:lang="en-US" payloadID="1615232861.000506@prd327utl1.int.coupahost.com" timestamp="2021-03-08T11:47:42-08:00">
+    <Header>
+        <From>
+            <Credential domain="NetworkID">
+                <Identity>Identity</Identity>
+            </Credential>
+        </From>
+        <To>
+            <Credential domain="TEST">
+                <Identity>Identity</Identity>
+            </Credential>
+        </To>
+    </Header>
+</cXML>
+----
+
+==== Source
+
+[source,dataweave]
+----
+%dw 2.0
+output application/json
+import * from dw::xml::Dtd
```

Comment:
TODO: check if the DW module name changed to `dw::xml::Dtd`?

Reply:
looks like it was `dw::util::Xml` at some point.

Reply:
TODO: double check

Reply:
added note to next format check PR: https://github.com/mulesoft/docs-dataweave/pull/225/files